stages/org.osbuild.ostree.deploy.container

```diff
@@ -92,7 +103,7 @@ def make_fs_identifier(desc):
 def ostree_container_deploy(tree, inputs, osname, target_imgref, kopts):
     images = containers.parse_containers_input(inputs)
     for image in images.values():
-        with containers.container_source(image) as (_, image_source):
+        with containers.container_source(image) as (image_name, image_source):
```

Comment:
@dustymabe We originally planned for the aleph file to be a new stage, but with the way that I've implemented it, I used `skopeo inspect` to get the data we need for the aleph. It's pretty simple to create a new stage, but it just means that we need to take in `images` as input again:
```
- type: org.osbuild.ostree.aleph
  inputs:
      images:
        type: org.osbuild.containers
        origin: org.osbuild.source
        mpp-resolve-images:
          images:
            - source: registry.gitlab.com/redhat/services/products/image-builder/ci/images/fedora-coreos
              tag: stable
```
I didn't like how we need to specify the exact same input as the container deploy stage which is why I just made it a part of the deploy stage, but what are your thoughts?

Reply:
At least with the existing aleph version bits in COSA today I thought we could just inspect the tree (filesystem) to get all the information we needed. These new changes obviously throw some complications in there. Not sure if we can get the new desired information from the filesystem too or not.

Reply:
I see... I think I'll need to do some investigation on that. If it is possible to get everything from the tree, is that preferred over using `skopeo inspect`?